plugin/contrib/mesos/pkg/scheduler/plugin_test.go

```diff
@@ -511,6 +515,9 @@ func TestPlugin_LifeCycle(t *testing.T) {
 	mockDriver.AssertNumberOfCalls(t, "LaunchTasks", 1)
 	assert.Equal(1, len(launchTasks_taskInfos))
 
+	// check that ExecutorInfo.data has the static pod data
+	assert.Len(launchTasks_taskInfos[0].Executor.Data, 3)
```

Comment:
I think we can do better here by testing the actual contents, not just the length.
```go
// Not sure what the right assert.Method to use here. Please adapt.
if got, want := launchTasks_taskInfos[0].Executor.Data, executor.Data; !bytes.Equal(got, want) {
    t.Fatalf("got: %v, want: %v", got, want)
}
```